tests/unit/core/reducers/test_categories.js

```diff
@@ -25,16 +25,11 @@ describe('categories reducer', () => {
     expect(loading).toEqual(false);
   });
 
-  it('defaults to not error', () => {
```

Comment:
I've mentioned this before, but I really like removing error state from the reducers. 👍